### PR TITLE
feat(http): HttpModule and JsonpModule

### PR DESCRIPTION
**Pull request by PatrickJS**
allow module support for nightly

### CODE REVIEW

File: modules/@angular/http/http.ts [hunk 1]
```diff
@@ -194,6 +194,13 @@ export const HTTP_PROVIDERS: any[] = [
   {provide: XSRFStrategy, useValue: new CookieXSRFStrategy()},
 ];
 
+/**
+ * The app module for http.
+ * @experimental
+ */
+@AppModule({providers: HTTP_PROVIDERS, directives: [], pipes: []})
```
Comment (gdi2290) [inline modules/@angular/http/http.ts]: fixed, thanks!

**Comment by gdi2290** [comment]
@vicb all green

**Comment by vicb** [comment]
@gdi2290 `AppModule` are still in flux - let's put this PR on hold for now, sorry about that.

**Comment by gdi2290** [comment]
@vicb I only needed this for nightly. If the name of `@AppModule` changes I'll make another PR if someone didn't globally rename it

**Comment by vicb** [comment]
We can not merge that now, changes are coming soon.

**Pull request closed without merging**